### PR TITLE
"Getting Started" page in ukrainian language

### PR DESCRIPTION
**Pull request by yarik-vv**
added "Getting Started" page in ukrainian language

### CODE REVIEW

**Comment by petetnt** [comment]
Cool stuff @yarik-vv! Happen to know anyone who knows ukrainian and would be willing to review this?

**Comment by FilipVoloshin** [comment]
@petetnt I can say that the translation is awesome. There are no mistakes in this article.

**Comment by yarik-vv** [comment]
@petetnt do you need this translation or close this pull request?

File: src/nls/urls.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ define(function (require, exports, module) {
         "sv": true,
         "zh-cn": true,
         "zh-tw": true,
-        "tr": true
+        "tr": true,
```
Comment (petetnt) [inline src/nls/urls.js]: Nit: these should be alphabetized, but the `tr` wasn't either so we can just fix that later

**Comment by petetnt** [comment]
Sorry @yarik-vv, this one feel through a crack and I managed to miss @FilipVoloshins comment! Merging now 👍

**Comment by petetnt** [comment]
Thanks for your first contribution to Brackets @yarik-vv, hopefully you will contribute again in the future!

**Pull request merged**